packages/components/organisms/f-checkout/test/component/f-checkout.component.shared.spec.js

```diff
@@ -15,8 +14,10 @@ describe('f-checkout component tests - @browserstack', () => {
         checkout.waitForComponent();
     });
 
-    it('should display the f-checkout component', () => {
+    it('should display the f-checkout component, @percy', () => {
```

Comment:
Out of curiosity, what does this `@percy` at the end do?

Reply:
We are using @percy tag to identify tests for visual regression. Not every tests needs to take screenshots, so only tests with this tag will run on Percy.

Reply:
Super nit, but maybe it's better to use `- @percy` as a suffix instead of the comma, since we might use the comma in a test name more regularly. Super nit :D

Reply:
This has been addressed in my supporting PR.